### ISSUE TITLE
Userscript doesn’t work for safari browser on iPad

### ISSUE THREAD

**Issue by krishnamohan276** [likeon/geometa#9]
Hello,

First of all, I want to thank you for this amazing tool to learn geoguessr meta. This works on macbook safari browser but not on iPad safari browser. 

I am using tamper monkey and the setup is exactly the same in both devices. Other userscripts work fine on both devices but this one doesn’t work on mobile devices. 

Is this expected? If not, I would really appreciate if you can fix the script so that it works on mobile devices too. 

Thanks

**Comment by likeon** [comment]
Hi,

We haven't specifically tested it on mobile devices, but just the other day, we received a bug report on our Discord about the userscript working unreliably on an iPad: it appeared to work initially but then stopped.

I’m afraid that, at this point, I can’t offer much help in resolving this issue because I don’t have an iPad myself.

**Comment by krishnamohan276** [comment]
I don't think it is iPad specific issue but it could be more of a mobile devices specific issue. It would have been great if it works on mobile devices too but it is fine, I understand. Thank you for the reply.

**Comment by krishnamohan276** [comment]
After updating the script to 0.76, the script started working on the safari browser of iPad. Thanks a lot!!
![IMG_0270](https://github.com/user-attachments/assets/031f5192-3c16-41da-9e0f-8960bc4fcef1)
